crates/stackable-operator/src/product_logging/framework.rs

```diff
@@ -755,8 +755,7 @@ sources:
   files_opa_bundle_builder:
     type: file
     include:
-      - {STACKABLE_LOG_DIR}/bundle-builder/current
-      - {STACKABLE_LOG_DIR}/bundle-builder/test
+      - {STACKABLE_LOG_DIR}/bundle-builder/opa-bundle-builder.log.json
```

Comment:
https://github.com/stackabletech/opa-operator/pull/578 migrates to this mechanism, see https://github.com/stackabletech/opa-operator/pull/578#discussion_r1611798885